### DISCUSSION TITLE
Would GC work if I create many  no-longer-used observables?

### DISCUSSION THREAD

**Discussion by gyhyfj** [ReactiveX/rxjs#7206] [Q&A] [answered]
For example, if I always create new observable on click or scroll by this callback:
```ts
const onClick =()=>of(1,2,3).subscribe(console.log)
```
Could garbage collector work to clean no longer used observable in browser?

**Comment by DoronTorangy** [comment, marked as answer]
Pretty sure once the observable stream is completed, it will be garbaged collected
